WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -254,7 +300,7 @@ struct AddEditCoupon: View {
             .toolbar {
                 ToolbarItem(placement: .cancellationAction) {
                     Button(Localization.cancelButton, action: {
-                        presentation.wrappedValue.dismiss()
+                        dismissHandler()
```

Comment:
Since we no longer dismiss the view with `presentation`, we should probably remove the declaration for this property above?

Reply:
Indeed! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/6889/commits/610110df70a7cd8dd178d2b8b9fb874b18b20c36